src/scripts.js

```diff
@@ -11,22 +12,65 @@ let userDailyStepGoal = document.querySelector('.user-daily-step-goal-js');
 let userFriends = document.querySelector('.user-friends-js');
 let stepGoalComparison = document.querySelector('.step-goal-comparison-js');
 let waterConsumedByDay = document.querySelector('.water-consumed-by-day-js');
-let waterConsumedByWeek = document.querySelector('.water-consumed-by-week-js')
+let waterConsumedByWeek = document.querySelector('.water-consumed-by-week-js');
+let hoursSlept = document.querySelector('.hours-slept-js');
+let sleepQuality = document.querySelector('.sleep-quality-js');
+let lastWeekSleep = document.querySelector('.last-week-sleep-js');
+let recomendedSleep = document.querySelector('.sleep-percentage-js')
+let avgSleepQuality = document.querySelector('.avg-sleep-quality-js')
+let avgHoursSleptAllTime = document.querySelector('.all-time-avg-hours-slept-js')
+let parentDiv = document.querySelector('.parent')
+
+parent.addEventListener('click', makeItShake);
+
+function makeItShake(event) {
+  if (event.target.classList.contains('tile')) {
+    event.target.closest('.tile').classList.toggle('vibe')
+  }
+}
 
 displayUserInfo(user);
+displayWaterConsumtion();
+displayStepInfo();
+displaySleepData();
 
 function displayUserInfo(user) {
   welcomeName.innerHTML = `${user.getFirstName()}`;
   userFullName.innerHTML = `${user.name}`;
   userAddress.innerHTML = `${user.address}`;
   userEmail.innerHTML = `${user.email}`;
+}
+
+
+function displaySleepData() {
+  avgHoursSleptAllTime.innerHTML = `${sleep.calculateTotalAvgHoursSlept()} hrs`
+  avgSleepQuality.innerHTML = `${(((sleep.calculateAvgSleepQuality())/5)*100).toFixed(2)}%`
+  recomendedSleep.innerHTML = `${sleep.calculatePercentageOfRecommendedSleep("2019/09/22")}`
+  sleepQuality.innerHTML = `${(((sleep.calculateQualityOfSleepByDay("2019/09/22"))/5)*100).toFixed(2)} %`;
+  hoursSlept.innerHTML = `${sleep.calculateHoursSleptByDay("2019/09/22")}`;
+  lastWeekSleep.innerHTML =
+  `<p><u><span>Date </span><span> : Hours Slept</span> : <span>Sleep Quality</span></u></p>
```

Comment:
I know this is beefy and we can possibly refactor it in the future but it works for now so once we have our MVP achieved we can look back on these.

Reply:
We can absolutely refactor this in the future. I have a pretty good idea on how we could do that with a nice forEach:-)